### PR TITLE
py-modred: New package

### PR DESCRIPTION
**Pull request by payerle**
Parallelized python module for finding modal decompositions and
reduced order models.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-modred/package.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    homepage = "http://github.com/belson17/modred"
+    git      = "https://github.com/belson17/modred.git"
+
+    version('2.0.4', tag='v2.0.4')
+    version('2.0.3', tag='v2.0.3')
+    version('2.0.2', tag='v2.0.2')
+    version('2.0.1', tag='v2.0.1')
+    version('2.0.0', tag='v2.0.0')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-modred/package.py]: Missing a dependency on `py-numpy`.

File: var/spack/repos/builtin/packages/py-modred/package.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    patch('v2x-setuptools-8.0.patch', when='@2: ^py-setuptools@8.0:')
+
+#    def build_args(self, spec, prefix):
+#        # FIXME: Add arguments other than --prefix
+#        # FIXME: If not needed delete this function
+#        args = []
+#        return args
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-modred/package.py]: You can delete this if you aren't going to use it.

File: var/spack/repos/builtin/packages/py-modred/package.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-modred/package.py]: You'll need to update this to the new license. See any recent file.

Comment (payerle) [reply var/spack/repos/builtin/packages/py-modred/package.py]: Requested changes made

1) fixed license
2) removed commented out code
3) added dependency on py-numpy

**Pull request merged**